braces/views/_ajax.py

```diff
@@ -56,30 +56,29 @@ def render_json_object_response(self, objects, **kwargs):
         return HttpResponse(json_data, content_type=self.get_content_type())
```

Comment:
why not  use django built in jsonresponse?

Reply:
I don't think it existed at the time when we wrote this mixin. Good call, though.

Reply:
I think I'll leave that refactor for later on, though. Odds are good we should just drop the mixin.